### PR TITLE
Da w refund

### PR DESCRIPTION
**Pull request by Jbern16**
## Description of the change

Adds DA with Refund 
>reminder: Any subgraph deployments should be documented as [Releases](https://github.com/ArtBlocks/artblocks-subgraph/releases) in this repository.

>reminder: Any changes to subgraph schema should be accompanied by corresponding changes to the Art Blocks documentation site, available at https://docs.artblocks.io/ (specifically, see the [Subgraph Entities](https://docs.artblocks.io/creator-docs/art-blocks-api/entities/) and [Subgraph Querying and API Overview](https://docs.artblocks.io/creator-docs/art-blocks-api/queries/) sections).

### CODE REVIEW

File: config/goerli-dev.json [hunk 1]
```diff
@@ -215,6 +215,13 @@
       "startBlock": 7828562
     }
   ],
+  "minterDAExpSettlementV0Contracts": [
+    {
+      "address": "0x0000000000000000000000000000000000000000",
```
Comment (Jbern16) [inline config/goerli-dev.json:220]: needs to be updated before merge

File: generated/schema.ts [hunk 2]
```diff
@@ -1317,6 +1334,23 @@ export class Account extends Entity {
...
+    if (!value || value.kind == ValueKind.NULL) {
+      return null;
+    } else {
+      return value.toStringArray();
+    }
+  }
+
+  set receipts(value: Array<string> | null) {
+    if (!value) {
+      this.unset("receipts");
```
[the remark this answers is not in the document]
Comment (Jbern16) [reply generated/schema.ts:1348]: not sure - I think so! These are generated files so I'm inclined to say its par for the course

[the remark this answers is not in the document]
Comment (lindsgil) [reply generated/schema.ts:1348]: 🙌 ty!

File: config/goerli-dev.json (hunk 1 above)
Comment (Jbern16) [reply config/goerli-dev.json:220]:
```suggestion
      "address": "0xaC85613A8C8b39a713175bC0F54630199A40D2D8",
```

**Review by lindsgil — APPROVED** [review]

**Pull request merged**